src/components/PassphraseForm.jsx

```diff
@@ -62,7 +62,7 @@ class PassphraseForm extends Component {
           key="current_password"
           value={currentPassword}
           inError={currentPasswordError}
-          onInput={e => this.handleCurrentInput(e)}
+          onChange={e => this.handleCurrentInput(e)}
```

Comment:
Why removing `onInput` ? `onInput` and `onChange` does not have the same behaviour.